### DISCUSSION TITLE
Make Loop Grid Widget Compatible With Other Popular WordPress Plugins

### DISCUSSION THREAD

**Discussion by joey-pk** [elementor/elementor#24529] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi,

I'm writing to report a potential compatibility issue between the new Elementor Loop Grid Widget and some popular WordPress plugins, including Ad Inserter Pro.

The problem seems to be that pages built with the Loop Grid Widget don't utilize the `the_content` WP filter hook. This means that plugins relying on that hook, such as Ad Inserter Pro, don't work properly on these pages.

Here are two screenshots demonstrating the issue:

Post Archive Widget: Image of post archive built with Elementor archive widget: https://prnt.sc/AjxDsdUQxMUy
Loop Grid Widget: Image of post archive built with Elementor loop grid widget: https://prnt.sc/yNF563PN3yG9

As you can see, Ad Inserter can identify insertion points on the page built with the archive widget but not on the one built with the loop grid widget. This suggests that the loop grid widget bypasses the `the_content` filter, causing compatibility problems.

Would you please advise on how to address this issue?

Thank you!

### Describe the solution you'd like

Update the loop grid widget so that it works just like the archive widget as far as the page structure is concerned.

### Describe alternatives you've considered

Configure custom action hooks to enable ad inserter pro determine insertion positions on pages built with the loop grid widget.

### Additional context

https://prnt.sc/AjxDsdUQxMUy

https://prnt.sc/yNF563PN3yG9

**Comment by jrstarenlard** [comment]
Hi,

Any updates on this issue?

It seems that the loop grid widget does use the the_content hook on other pages, but if you use it on an 'archive' type template it doesn't. Why?
